### ISSUE TITLE
Configuración del mapa (título)

### ISSUE THREAD

**Issue by ChemiF1** [ChemiF1/Motorsport-Software-Solutions-by-Data-Performance#3]
### DESCRIPCIÓN

Una vez que tenemos esto en marcha lo siguiente que tenemos que hacer es trabajar en la configuración, para hacerlo más accesible y fácil de manejar por su parte y de los posibles clientes que podamos encontrar

Para esto las cosas que tenemos que conseguir son: 

- [x] marcar la distancia de cada tramo: Manel quiere que pueda tener capacidad de poder decidir cuanto mide cada zona del tramo, de manera que pueda ser de 1km, 2km, 5km, 0.8km... lo que se pueda necesitar

- [x] quitar los ejes: en la versión que tenemos en primera instancia hemos podido sacar esto
![image](https://github.com/user-attachments/assets/336be17f-de33-435c-8b0e-f2dd97a74b9e)
No es necesario mostrar los ejes, porque no dicen casi nada

- [x] meter la leyenda dentro del mapa: para aprovechar mejor el espacio creo que se podría probar a meter la leyenda dentro del gráfico, como hacemos con la portada de tiempos en F1
![Portada](https://github.com/user-attachments/assets/c4f02c2a-bbdd-4b1b-8ecf-b8b74ab4bbdb)

- [x] poder configurar el nombre de título: esto es exactamente igual que lo que hacemos en F1, que podamos poner nosotros el título, o si no se coloca nada que se quede el nombre original del tramo
![image](https://github.com/user-attachments/assets/e1ed6e0b-2bbf-4319-8ceb-b85839558cd2)

- [x] reducir el tamaño de la leyenda: no es necesario que la leyenda ocupe todo el alto, con que quede como en la imagen de tiempos de F1 basta y sobra. Ya se irá añadiendo a medida que se vaya expandiendo

**Comment by LuisOme** [comment]
Te dejo aquí lo que he conseguido de momento:

Para tramos, podemos coger un valor X para que todos los tramos en ese dibujo sean de igual forma.
Por ejemplo 500 metros y que todos los trozos sean de 500 metros:
![image](https://github.com/user-attachments/assets/f126af9c-b7c4-4309-9c48-7462de6ef290)

Puesto un Nombre de Titulo_Grafico y Titulo_Leyenda para marcar ambas:
![image](https://github.com/user-attachments/assets/d6cec3ca-7fd4-497a-88f1-7f49683d98ad)
![image](https://github.com/user-attachments/assets/d20a6af8-d17f-4e66-a88b-952128d3a9e1)

Meter la leyenda dentro del mapa me parece que en muchos casos no va a quedar clara porque si son tramos muy largos puede quedar inmensa, creo que es mejor dejarla fuera, pero he quitado la barra de colores que ocupa bastante.
Debajo de la leyenda supongo que se podría meter el texto con los daatos que se quieran.


CREO que faltaría poder coger trozos arbitrarios por tramo, es decir, que por ejemplo si tengo una seccion de 5 kms se coja como 500 metros/2000 metros / 1500 metros / 1000 metros
Esto lo veo más complicado, sobre todo a la hora de que se pueda automatizar, pero sería crear una especie de lista, pero habría que asegurarse que la suma coincida con los kms totales, si no dara algun error que desconozco.

**Comment by ChemiF1** [comment]
Reabro el issue para que lo pueda seguir Álvaro

**Comment by afisicos** [comment]
Una duda. ¿Estas imágenes de los plots de circuito salen del programa en python que teníais?

**Comment by ChemiF1** [comment]
Sí, lo que veas de F1 (o de circuito cerrado) es de nuestro código

**Comment by afisicos** [comment]
Me refiero a las de circuito abierto de este mismo issue

**Comment by ChemiF1** [comment]
También, eso salía del script que hizo

**Comment by afisicos** [comment]
![Image](https://github.com/user-attachments/assets/e00a74b3-a13e-44a7-abf5-fb3cc93e795f)

Primeros éxitos cargando el KML en Unity. 

Estoy cogiendo los nodos que conforman un tramo y poniéndolos sobre la escena. 
Lo siguiente es unirlos formando una carretera visible. Que no me costará mucho.

Una vez consiga esto, dejaré una función hecha para mostrar un tramo en concreto, de modo que sea fácil para Luis llamar a esa función y que pinte el tramo que elija. Y así no tenga que meterse al código de esa función a fondo. 

Tras las pruebas que llevo, creo que lo mejor será hacerlo en Unity. Le veo más posibilidades que a otras tecnologías y tengo más conocimiento.

**Comment by afisicos** [comment]
Ahora mismo las etiquetas sobre el mapa son automáticas y se colocan en cada KM justamente.
Habrá que desarrollar el poder ponerlas a la altura que queramos.

Por otro lado, las secciones sí se pueden indicar manualmente y se definen entre 2 números decimales (distancia de inicio y distancia de fin)

**Comment by ChemiF1** [comment]
Aquí nos podría faltar la opción de poder configurar el nombre del tramo, en el caso de que lo quisiera colocar el usuario

![Image](https://github.com/user-attachments/assets/8a7133a2-f36b-4507-a27d-831166e171e4)

**Comment by afisicos** [comment]
![Image](https://github.com/user-attachments/assets/306d4680-04d7-4c74-abe3-90d88adb3b1c)

**Comment by ChemiF1** [comment]
Todo esto lo hemos dejado ya terminado en las ultimas versiones, se puede modificar el nombre como se quiera

![Image](https://github.com/user-attachments/assets/1165d51f-1ae9-4bdd-b456-eda66a463136)
